Sources/Drill/TableFormatter.swift

```diff
@@ -0,0 +1,93 @@
+/// TableFormatter.swift
+///
+/// Copyright 2017, The Silt Language Project.
+///
+/// This project is released under the MIT license, a copy of which is
+/// available in the repository.
+
+import Foundation
+
+/// Represents a column of tabular values, including a title and rows.
+struct Column {
+  /// The column's title.
+  let title: String
+
+  /// the rows in the column.
+  var rows = [String]()
+
+  /// Creates an empty column with the provided title.
+  init(title: String) {
+    self.title = title
+  }
+
+  /// The length of the longest line in this column.
+  var width: Int {
+    var longest = self.title.count
+    for row in self.rows where row.count > longest {
+      longest = row.count
+    }
+    return longest
+  }
+}
+
+/// An object that formats a set of columns to an output stream as an ASCII
+/// table.
+enum TableFormatter {
+  /// Writes the provided columns as an ASCII table to the provided output
+  /// stream. It takes into account the lengths of the columns and ensures a
+  /// perfectly-formatted box.
+  /// Example:
+  /// ```
+  /// ┏━━━━━━━━━━━━━┳━━━━━━━━━┓
+  /// ┃ Pass        ┃ Time    ┃
```

Comment:
I assume this isn't rendering right in my browser...

Reply:
GitHub adds more spaces than terminal emulators.

Reply:
<img width="1089" alt="screen shot 2017-11-29 at 3 12 10 pm" src="https://user-images.githubusercontent.com/853032/33396813-b4860fa8-d517-11e7-8289-134be54f57bf.png">